esileapclient/tests/functional/utils/output_utils.py

```diff
@@ -0,0 +1,40 @@
+from tempest.lib.cli import output_parser
+
+
+def parse_details(output):
+    """ Takes a two-column table with "Field" and "Value" headers from the
```

Comment:
Is a table simply a string that's eventually parsed by the tempest parser into a dict?

Reply:
yep. when you create something or show the details of something (project, offer, user, etc.) it'll return a table that looks like this:
```
+---------------------+----------------------------------+
| Field               | Value                            |
+---------------------+----------------------------------+
| domain_id           | default                          |
| enabled             | True                             |
| id                  | 818b854ca26f4072b34531c89040475a |
| name                | test-user                        |
| options             | {}                               |
| password_expires_at | None                             |
+---------------------+----------------------------------+

```

Reply:
OpenStack commands have a variety of non-tabular output formats (including JSON) that would make `parse_details` unnecessary.

Reply:
I didn't even think to parse the output as JSON-- since I'll be updating the implementation of the dummy node class to output a JSON-ified python dict to a file, I'll also update this to pull the output of commands with the `-f json` flag into a python dict instead of using the parser.